app/routes/docs.readme.mdx

```diff
@@ -0,0 +1,56 @@
+---
+meta:
+  - title: Welcome to the metadata catalog
+---
+
+## Welcome to the metadata catalog
+
+Nice to see that you would like to want to make your project data findable!
+
+The purpose of this metadata catalog 
+
+- To create clarity and transparency about the data your project used and generated
+- To enable you register your project metadata so that other researchers can find it later on, when they are working on a relevant topic
```

Comment:
- Create clarity and transparency about the data your project used and generated
- Enable you to register your project metadata so that other researchers can find it later on, when they are working on a relevant topic